app/marketing/views.py

```diff
@@ -885,6 +886,25 @@ def trending_quests():
         ).order_by('-recent_attempts').all()[0:10]
     return quests
 
+def quest_of_the_day():
+    quest = trending_quests()[0]
+    return quest
+
+def upcoming_grant():
+    cutoff_date = timezone.now() - timezone.timedelta(days=7)
+    grant = Grant.objects.filter(created_on__gte=cutoff_date).order_by('-created_on')[:5][0]
+    return grant
+
+def upcoming_hackathon():
+    cutoff_date = timezone.now() - timezone.timedelta(days=7)
+    hackathon = HackathonEvent.objects.filter(created_on__gte=cutoff_date).order_by('-created_on')[:5][0]
+    return hackathon
+
+def latest_activities():
+    cutoff_date = timezone.now() - timezone.timedelta(days=7)
+    activities = Activity.objects.filter(created_on__gte=cutoff_date).order_by('-created_on')[:4]
```

Comment:
i think you should probably only show posts that are townsuqare worthy.

retail.views:get_specific_activities('connect', 0, user)

Reply:
Ok. Will try using `activities = get_specific_activities('connect', 0, user)`

Reply:
This is done in [ba68b4a](https://github.com/gitcoinco/web/pull/6471/commits/ba68b4a5d02bf47f0d6b6492d758b7809dcc12e1)